doc/index.html

```diff
@@ -162,15 +162,10 @@ <h2>Security <a href="#security">&#x1F517;</a></h2>
             </section>
 
             <section id="lint">
-                <h2>Linting and hinting <a href="#lint">&#x1F517;</a></h2>
-                <p>We <em>lint</em> our code using <a href="https://www.npmjs.com/package/eslint"><code>eslint</code></a>: see
-                    <a href="https://github.com/w3c/echidna/blob/master/package.json#L81">an example of
-                    usage in Echidna</a>, and <a href="https://github.com/w3c/echidna/blob/master/.eslintrc.json">the config file we maintain</a>.</p>
-                <p>The package <a href="https://www.npmjs.com/package/eslint-plugin-node"><code>eslint-plugin-node</code></a> is also used, to provide a
-                    baseline of common rules (which we modify slightly via config).</p>
-                <p>We <em>hint</em> our code using <a href="https://www.npmjs.com/package/jshint"><code>jshint</code></a>: see
-                  <a href="https://github.com/w3c/Eunomia/blob/master/package.json#L45">an example of
-                    usage in Eunomia</a>, and <a href="https://github.com/w3c/Eunomia/blob/master/.jshintrc">the config file we maintain</a>.</p>
+                <h2>Linting<a href="#lint">&#x1F517;</a></h2>
+                <p>We <em>lint</em> our code using <a href="https://www.npmjs.com/package/eslint"><code>eslint</code></a> with <a href="https://www.npmjs.com/package/eslint-config-airbnb-base">eslint-config-airbnb-base</a> code style, see <a href="https://github.com/w3c/spec-generator/blob/864ecd2542eb59ff7cab4299e1e813153dfd4ac0/package.json#L33">an example of usage</a> in spec-generator, and the <a href="https://github.com/w3c/spec-generator/blob/main/.eslintrc.json">eslint config file</a> we maintain.</p>
```

Comment:
https://github.com/w3c/specberus/blob/main/.eslintrc.json might be a better example as it's not using the legacy style

Reply:
Shouldn't a base `.eslintrc.json` file be put in this `w3c/nodejs` repo instead of referencing a copy used in another project?

Reply:
I point to spec-generator main branch, because it'll be more up-to-date. The copy, as @vivienlacourba suggested, would be out-dated. 

We can also remove legacy for spec-generator, what do you think, @deniak ?

Reply:
> I point to spec-generator main branch, because it'll be more up-to-date. The copy, as @vivienlacourba suggested, would be out-dated.

Yes, I agree it's better to point to a config file we know won't be out-dated.

> We can also remove legacy for spec-generator, what do you think, @deniak ?

+1